dozer-orchestrator/src/cli/types.rs

```diff
@@ -2,6 +2,7 @@ use clap::{Args, Parser, Subcommand};
 
```

Comment:
We can put `struct Cloud` and relavant `use` and `struct` in a module and reduce two `#[cfg(feature = "cloud")]`